src/components/collection/summary/SummaryHeader.tsx

```diff
@@ -66,22 +75,44 @@ const SummaryHeader: React.FC<SummaryHeaderProps> = ({
     (itemLike: ItemLikeRecord) => itemLike?.item.id === collection?.id,
   );
 
+  const openLoginSnackbarMessage = () => {
+    setOpen(true);
+  };
+
+  const handleCloseSnackBarMessage = (
+    event: React.SyntheticEvent | Event,
+    reason?: string,
+  ) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+
+    setOpen(false);
+  };
   const handleLike = () => {
-    if (!collection?.id || !member?.id) {
+    if (!collection?.id) {
       console.error('unable to like an item which id is undefined');
       return;
     }
+    if (!member?.id) {
+      openLoginSnackbarMessage();
+      return;
+    }
     postItemLike({
       itemId: collection?.id,
       memberId: member.id,
     });
   };
 
   const handleUnlike = () => {
-    if (!collection?.id || !member?.id) {
+    if (!collection?.id) {
       console.error('unable to unlike an item which id is undefined');
       return;
     }
+    if (!member?.id) {
+      openLoginSnackbarMessage();
+      return;
+    }
```

Comment:
In my opinion it is very unlikely that a user could unlike an item if they are not logged in. But we can keep it like this.